.gitignore

```diff
@@ -93,6 +93,10 @@ scripts/nodes.txt
 # OS X folder attributes
 .DS_Store
 
+# IPython
```

Comment:
@robertnishihara Thoughts on these additions? I don't think they're necessary, but I'll let you make the call

Reply:
Seems fine. If it's an issue we can always revert the changes to the `.gitignore`.